src/libraries/Native/Windows/Common/pal_safecrt.h

```diff
@@ -0,0 +1,56 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
All this duplication is unfortunate. 

You may want to break this change into smaller steps that are more acceptable. 

We have more and more of the Unix libraries coming to Windows. Compression was on Windows in forever, Globalization is over (cc @safern) and now you would like to have build crypto (as optional non-shipping test-only). So the first step may be to get rid of the Windows/Unix separation under libraries/Native and more everything to the same level.